CONTRIBUTING.md

```diff
@@ -11,6 +12,21 @@ First of all, thanks for taking the time to contribute! 🎉👍
   - [Errors handling](#errors-handling)
 
 ---
+
+## New instance naming
+
+In case you want to create a new instance on the `OpenTermsArchive` organization, please choose an instance name as descriptive as possible.
+
+Here are some examples of existing instances and the kind of declaration files they contain:
```

Comment:
I think the point of this documentation should be to explain what makes an instance “descriptive” or “understandable”, not so much to insist on the fact that it should be 😉 The example we give below do not explain in which way they are descriptive.